model/src/main/proto/spine_examples/chatspn/chat/commands.proto

```diff
@@ -68,3 +68,16 @@ message CreateGroupChat {
     // The name of the chat to create.
     string name = 4 [(required) = true];
 }
+
+// Tells to include a new members to the chat.
+message IncludeMembers {
```

Comment:
Will we have a button with a caption "Include members"?

I think, the button will be "Add members". Hence, the command should sound the same.

Reply:
I took this name based on the suggestion that in the future the `MebersAddition` process will appear, then it will have the `AddMembers` command.